### DISCUSSION TITLE
Dump all events in a single file?

### DISCUSSION THREAD

**Discussion by saumier** [culturecreates/artsdata-planet-lavitrine#18] [Data Sources] [answered]
Currently I have setup multiple dump files (artifacts) for GTQ, PDA, the dia-log sites and a couple of additional sites outside of dia-log. 

Would LaVitrine prefer a single dump file for all events?  

Note: I will keep people, places and organizations in separate dump files.

**Comment by saumier** [comment]
@christianroy What are your thoughts on a single event dump instead of multiple?

**Reply by christianroy** [reply]
C'est probablement plus simple d'un point de vue opérationnel, donc ce pourrait être une bonne idée du point de vue du script.

D'un point de vue humain pour faire des diagnostic ou comprendre la nature des données, c'est quand même avantageux d'avoir des différents dumps pour comprendre les différentes sources.

Donc opinion neutre... ou légère préférence pour laisser as-is.

**Reply by christianroy** [reply, marked as answer]
Après discussion, on confirme qu'on peut faire un seul dump comprenant toutes les sources.

**Reply by saumier** [reply]
I have created issue https://github.com/culturecreates/artsdata-planet-lavitrine/issues/23 for tracking this work.
